src/types.ts

```diff
@@ -132,9 +132,12 @@ export interface Service {
 
 	ports?: ListOfUniqueItems<string>;
 
+	platform?: string;
```

Comment:
@toochevere is there any reason we would want to allow setting the target platform of a service via composefile?

Reply:
this needs to be handled properly in https://github.com/balena-io-modules/balena-multibuild if we decide to keep it

Reply:
@robertgzr I think that the only issue is how it will potentially interact with the `--platform` info we are potentially passing in with the new multi-arch work 🤔 .   Also, what would be the use case for target platform via compose file?  Isn't everything in a compose file always going to be the same platform?

Reply:
@toochevere does `balena.yml` support specifying which platforms an application can target? other than that I don't see anything this could be useful for...

Reply:
let's postpone this for now... technically the builder/cli can decide to handle this field (or not) just like with the other build related flags...